addons/website_sms/views/website_visitor_views.xml

```diff
@@ -20,11 +20,6 @@
             <field name="country_id" position="after">
                 <field name="mobile" widget="phone"/>
             </field>
-            <xpath expr="//div[@name='w_visitor_kanban_actions']" position="inside">
-                <button name="action_send_sms" type="object" class="btn btn-secondary"
-                        invisible="not mobile">SMS
-                </button>
-            </xpath>
             <xpath expr="//div[hasclass('w_visitor_kanban_actions_ungrouped')]" position="inside">
```

Comment:
Again, everything uses the same display now, there is no more "w_visitor_kanban_actions_ungrouped", let's remove that class and adapt overrides please.